scripts/test/shared.py

```diff
@@ -186,6 +186,21 @@ def is_exe(fpath):
                                          'wasm-emscripten-finalize')]
 BINARYEN_JS = os.path.join(options.binaryen_root, 'out', 'binaryen.js')
 
+BINARYEN_JS_TEST_DIR = os.path.join(options.binaryen_test, 'binaryen.js')
+CRASH_TEST_DIR = os.path.join(options.binaryen_test, 'crash')
+CTOR_EVAL_TEST_DIR = os.path.join(options.binaryen_test, 'ctor-eval')
+EXAMPLE_TEST_DIR = os.path.join(options.binaryen_test, 'example')
+LLD_TEST_DIR = os.path.join(options.binaryen_test, 'lld')
+METADCE_TEST_DIR = os.path.join(options.binaryen_test, 'metadce')
+PASSES_TEST_DIR = os.path.join(options.binaryen_test, 'passes')
+PRINT_TEST_DIR = os.path.join(options.binaryen_test, 'print')
+REDUCE_TEST_DIR = os.path.join(options.binaryen_test, 'reduce')
+SPEC_TEST_DIR = os.path.join(options.binaryen_test, 'spec')
+UNIT_TEST_DIR = os.path.join(options.binaryen_test, 'unit')
+VALIDATOR_TEST_DIR = os.path.join(options.binaryen_test, 'validator')
+WASM2JS_TEST_DIR = os.path.join(options.binaryen_test, 'wasm2js')
+WASM_BACKEND_TEST_DIR = os.path.join(options.binaryen_test, 'wasm_backend')
```

Comment:
I'm not a big fan of all these global, and especially not of the giant import statement including them all.

How about exposing a single function called `get_test_dir`:

```
def get_test_dir(name):
  return os.path.join(options.binaryen_test, 'passes')
```

Then for example that only have one use you would simply have `get_test_dir('passes')`?

Reply:
Good idea! Done.